<define-tag pagetitle>DSA-1905-1 python-django</define-tag>
<define-tag report_date>2009-10-10</define-tag>
<define-tag secrefs>Bug#550457</define-tag>
<define-tag packages>python-django</define-tag>
<define-tag isvulnerable>yes</define-tag>
<define-tag fixed>yes</define-tag>

#use wml::debian::security

<h3>Debian GNU/Linux 5.0 (lenny)</h3>

<dl>

<dt><source />

  <dd><fileurl http://security.debian.org/pool/updates/main/p/python-django/python-django_1.0.2-1+lenny2.dsc />
  <dd><fileurl http://security.debian.org/pool/updates/main/p/python-django/python-django_1.0.2.orig.tar.gz />
  <dd><fileurl http://security.debian.org/pool/updates/main/p/python-django/python-django_1.0.2-1+lenny2.diff.gz />

<dt><arch-indep />

  <dd><fileurl http://security.debian.org/pool/updates/main/p/python-django/python-django_1.0.2-1+lenny2_all.deb />

</dl>

<p><md5sums http://lists.debian.org/debian-security-announce/2009/msg00227.html /></p>
